libs/constrained-generators/src/Constrained/Base.hs

```diff
@@ -3330,7 +3360,7 @@ genFromFold (nub -> must) (simplifySpec -> size) elemS fn foldS =
         withMode Loose $
           ( withMode m $
               suchThatWithTryT
-                1000
+                500
```

Comment:
??

Reply:
This was because one test was timing out. I thought this might be causing it. But Neil thinks it is something else.
I will put it back to 1000.